scripts/gitlab/build-unix.sh

```diff
@@ -43,11 +48,6 @@ fi
 echo "_____ Calculating checksums _____"
 for binary in $(ls)
 do
-  rhash --sha256 $binary -o $binary.sha256
-  if [ "${CARGO_TARGET}" = "armv7-linux-androideabi" ]
-  then
-    echo "> ${binary} cannot be hashed with cross-compiled binary"
-  else
-    ./parity tools hash $binary > $binary.sha3
-  fi
+  rhash --sha256 $binary -o $binary.sha256 #do we still need this hash (SHA2)?
```

Comment:
SHA256 is used for Vanity Service and Homebrew (Releases) And One-line Installer

Reply:
:+1: